spec/system/searching/filter_by_date_range_spec.rb

```diff
@@ -10,8 +10,8 @@
     visit '/'
     click_link 'Search'
 
-    fill_in 'search-application-search-filter-submitted-after-field', with: after_date
-    fill_in 'search-application-search-filter-submitted-before-field', with: before_date
+    fill_in 'filter-submitted-after-field', with: after_date
```

Comment:
I line the pattern of having these in `lets` its nice to have consistancy across the specs if we can.